core/sql/executor/ExFastTransport.h

```diff
@@ -407,6 +408,7 @@ class ExHdfsFastExtractTcb : public ExFastExtractTcb
                           
   NABoolean isSequenceFile();
   void createSequenceFileError(Int32 sfwRetCode);
+  void createHdfsClientFileError(Int32 sfwRetCode);
```

Comment:
should be hdfsClientRetCode instead of sfwRetCode

Reply:
Will do in my next commit though it is a just parameter name in the method declaration.